src/Business Foundation/App/NoSeries/src/Single/NoSeries.Codeunit.al

```diff
@@ -326,6 +326,18 @@ codeunit 310 "No. Series"
         exit(NoSeriesImpl.GetNoSeriesLine(NoSeriesLine, NoSeriesCode, UsageDate, HideErrorsAndWarnings));
     end;
 
+    /// <summary>
+    /// Use this method to check if number series is marked for date order.
+    /// </summary>
+    /// <param name="NoSeriesCode">The No. Series code to check.</param>
+    /// <returns>True if the number series is marked for date order, false otherwise.</returns>
+    procedure IsNoSeriesInDateOrder(NoSeriesCode: Code[20]): Boolean
```

Comment:
I'd add an overload that takes in the No. Series record as well. If you already have the record there is no need to Get it again.